fastlane/Fastfile

```diff
@@ -230,55 +230,6 @@ platform :ios do
   # Release Lanes
   #####################################################################################
 
-  # Executes the code freeze
-  #
-  # - Call start_code_freeze
-  # - Call complete_code_freeze
-  # - Go over PRs with the current milestone and add a comment to move them to the next
-  # - Open a PR targeting trunk
-  # - Send a message to Slack channel confirming the code freeze was finished
-  #
-  # @option [Boolean] skip_confirm (default: false) If true, avoids any interactive prompt
-  #
-  desc 'Executes all the code freeze steps'
-  lane :code_freeze do |options|
-    start_code_freeze(options)
-    comment_on_prs(milestone: release_version_current)
-    complete_code_freeze(options)
-  end
-
-  # Go through PRs and check their milestones, leaving a message if the target milestone is being frozen
-  def comment_on_prs(milestone:)
-    require_env_vars!('GITHUB_TOKEN')
-
-    version = milestone
-    UI.message("Checking open PRs pointing to #{version} milestone...")
-
-    github_api(
-      server_url: 'https://api.github.com',
-      api_token: get_required_env!('GITHUB_TOKEN'),
-      http_method: 'GET',
-      path: "/repos/#{GHHELPER_REPO}/pulls",
-      error_handlers: {
-        '*' => proc do |_result|
-          UI.important('⚠️ Failed checking open PRs, skipping this step.')
-        end
-      }
-    ) do |result|
-      result[:json].each do |pr|
-        next if pr['milestone'].nil?
-        next unless pr['milestone']['title'].include?(version)
-
-        comment_on_pr(
-          project: GHHELPER_REPO,
-          pr_number: pr['number'],
-          body: "\nThis PR is targeting #{version} which is entering code freeze. If this needs to be included in the #{version} release please merge it into `release/#{version}` if not please move it to the next milestone.",
-          reuse_identifier: 'code-freeze-in-progress'
-        )
-      end
-    end
-  end
```

Comment:
This custom implementation was replaced by a call to our `update_assigned_milestone` action from `release-toolkit` instead.

The call to `update_assigned_milestone` is now wrapped inside [the `def update_milestone` helper function below](https://github.com/Automattic/pocket-casts-ios/pull/2215/files#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688R1042-R1067), which takes care of also adding the frozen marker ❄️  to the milestone after having moved the PRs, and handling potential exceptions (milestone already updated)